konduit-serving-test/src/test/java/ai/konduit/serving/verticles/python/custom/PythonArrowInputTest.java

```diff
@@ -0,0 +1,146 @@
+/*
+ *
+ *  * ******************************************************************************
+ *  *  * Copyright (c) 2015-2019 Skymind Inc.
+ *  *  * Copyright (c) 2019 Konduit AI.
+ *  *  *
+ *  *  * This program and the accompanying materials are made available under the
+ *  *  * terms of the Apache License, Version 2.0 which is available at
+ *  *  * https://www.apache.org/licenses/LICENSE-2.0.
+ *  *  *
+ *  *  * Unless required by applicable law or agreed to in writing, software
+ *  *  * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  *  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  *  * License for the specific language governing permissions and limitations
+ *  *  * under the License.
+ *  *  *
+ *  *  * SPDX-License-Identifier: Apache-2.0
+ *  *  *****************************************************************************
+ *
+ *
+ */
+
+package ai.konduit.serving.verticles.python.custom;
+
+import ai.konduit.serving.InferenceConfiguration;
+import ai.konduit.serving.config.ServingConfig;
+import ai.konduit.serving.miscutils.PythonPathInfo;
+import ai.konduit.serving.model.PythonConfig;
+import ai.konduit.serving.pipeline.step.PythonStep;
+import ai.konduit.serving.verticles.inference.InferenceVerticle;
+import ai.konduit.serving.verticles.numpy.tensorflow.BaseMultiNumpyVerticalTest;
+import com.jayway.restassured.specification.RequestSpecification;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Handler;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import org.datavec.api.split.FileSplit;
+import org.datavec.api.split.partition.NumberOfRecordsPartitioner;
+import org.datavec.api.transform.schema.Schema;
+import org.datavec.api.writable.NDArrayWritable;
+import org.datavec.arrow.ArrowConverter;
+import org.datavec.arrow.recordreader.ArrowRecordWriter;
+import org.datavec.arrow.recordreader.ArrowWritableRecordBatch;
+import org.datavec.python.PythonType;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nd4j.linalg.factory.Nd4j;
+import org.nd4j.linalg.io.ClassPathResource;
+import org.nd4j.linalg.primitives.Pair;
+
+import javax.annotation.concurrent.NotThreadSafe;
+import java.io.File;
+import java.util.Collections;
+
+import static com.jayway.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.Matchers.isEmptyOrNullString;
+
+@RunWith(VertxUnitRunner.class)
+@NotThreadSafe
+@Ignore
```

Comment:
If these tests are expected to be ignore forever then better just remove them

Reply:
As Arrow Input test we are facing runtime error while inference so we made it as Ignore at the time, need to fix in framework and accordingly need to fix test class.
other ignore  for methods like classification and regression some output formats are not supported , so if not needed in future also we will remove it.

Reply:
As Arrow Input test we are facing runtime error while inference so we made it as Ignore at the time, need to fix in framework and accordingly need to fix test class.
other ignore  for methods like classification and regression some output formats are not supported , so if not needed in future also we will remove it.